reactive/vpe_router.py

```diff
@@ -25,7 +25,7 @@
 @hook('config-changed')
 def validate_config():
     try:
-        if ('pass', 'vpe-router', 'user') not in cfg:
+        if not cfg.keys() & {'pass', 'vpe-router', 'user'}:
```

Comment:
yeah, I see that. I'd prefer this instead:

```
>>> cfg
{'vpe-router': None, 'obama': None, 'user': None, 'pass': None}
>>> ('pass', 'vpe-router', 'user') in cfg.keys()
False
>>> ['pass', 'vpe-router', 'user'] in cfg.keys()
False
>>> all(k in cfg for k in ['pass', 'vpe-router', 'user'])
True
>>> del cfg['pass']
>>> cfg
{'vpe-router': None, 'obama': None, 'user': None}
>>> all(k in cfg for k in ['pass', 'vpe-router', 'user'])
False
```

so

`if all(k in cfg for k in ['pass', 'vpe-router', 'user']):`